client/state/action-types.js

```diff
@@ -29,7 +29,11 @@ export const ANALYTICS_PAGE_VIEW_RECORD = 'ANALYTICS_PAGE_VIEW_RECORD';
 export const ANALYTICS_STAT_BUMP = 'ANALYTICS_STAT_BUMP';
 export const AUTOMATED_TRANSFER_ELIGIBILITY_REQUEST = 'AUTOMATED_TRANSFER_ELIGIBILITY_REQUEST';
 export const AUTOMATED_TRANSFER_ELIGIBILITY_UPDATE = 'AUTOMATED_TRANSFER_ELIGIBILITY_UPDATE';
+export const AUTOMATED_TRANSFER_INITIATE = 'AUTOMATED_TRANSFER_INITIATE';
```

Comment:
I wonder if this is overlapping with the work that Hyperion was doing on themes? It seems they are using separate actions for theme transfer ([more info](https://github.com/Automattic/wp-calypso/blob/17f1cb9b7575b905c4fe3960c3414294038bb5c1/client/state/themes/upload-theme/reducer.js)), so maybe we should define specific ones for plugin transfer too?

On the other hand, it might be preferable to use generic ones in both cases.